build.gradle

```diff
@@ -121,12 +122,15 @@ uploadArchives {
 			beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 			
 			// upload to sonatype OSS (snapshot/release)
+			/*
 			repository(url: this.version.endsWith('-SNAPSHOT') ?
 					'https://oss.sonatype.org/content/repositories/snapshots' :
 					'https://oss.sonatype.org/service/local/staging/deploy/maven2') {
 				authentication(userName: this.hasProperty('sonatypeUsername') ? sonatypeUsername : '',
 					password: this.hasProperty('sonatypePassword') ? sonatypePassword : '')
 			}
+			*/
+			repository(url: mavenLocal().url)
```

Comment:
That part of the file needs to be reverted I think - deployments should be done to MavenCentral/Sonatype OSS.
I guess that was just for test purposes? Doing `./gradlew install` should also do the trick to install it into the local Maven repository.